packages/e2e-tests/specs/experiments/navigation-editor.test.js

```diff
@@ -204,7 +204,13 @@ describe( 'Navigation editor', () => {
 		);
 		await addAllPagesButton.click();
 
+		await page.waitForXPath( '//a[contains(., "Sample Page")]' );
```

Comment:
Previously this test mocked the pages REST API endpoint (`getPagesMocks`) to ensure it had consistent results.

While it may not be a problem right now, the test can't rely on 'Sample Page' being there, another test might remove it or change the title.

But we can't do API mocking either on the Page List block as it uses `get_pages`.

Not really sure what we can do to get the most out of this test. Maybe we should just go back to only the snapshot as the assertion, at least that checks the right block is inserted.